public/static/docs/command-reference/init.md

```diff
@@ -67,3 +70,28 @@ $ cat .dvc/.gitignore
 ...
 /cache
 ```
+
+Create a new <abbr>DVC repository</abbr> in a subdirectory of Git repository:
+
+```dvc
+$ mkdir repo && cd repo
+
+$ git init
+$ mkdir subrepo && cd subrepo
+
+$ dvc init --subdir
+```
+
+In this case, Git repository is inside `repo` directory, while <abbr>DVC
+repository</abbr> is inside `repo/subrepo`.
```

Comment:
The Git repo is in the `repo/` directory, while the _subdir_ <abbr>DVC repository</abbr> lives in `repo/subdir`.